calculator/src/App.jsx

```diff
@@ -0,0 +1,81 @@
+/** @jsxImportSource @emotion/react */
+import { useState } from "react";
+import { css } from "@emotion/react";
+import "./App.css";
+import CalculatorActions from "./CalculatorActions.jsx";
+import CalculatorDigits from "./CalculatorDigits.jsx";
+
+function App() {
```

Comment:
Do you really need a component which sole purpose is to render another component?